### PR TITLE
Updated test imports

### LINKED ISSUE #18860: Switch OIM imports to NIM in tests
## Definition of Ready

- #18858 must be resolved.

## Definition of Done

- Imports in all packages are converted.
- ESLint rule detects a relative (cross-package) import and reports an error.

## Procedure per package

> [!NOTE]
> If a private API is required in a test, we need to decide if the module should still be marked as private,

- Uncomment the exclusion related to a package.
- Resolve ESLint issues.
- Imports within the same package must be relative.
- Cross-package CKEditor 5 imports must use an entry point.

> [!WARNING]
> Each pull request should convert only a single package.

## Packages to cover

**CKEditor 5:**

- [ ] ckeditor5
- [ ] ckeditor5-adapter-ckfinder
- [ ] ckeditor5-alignment
- [ ] ckeditor5-autoformat
- [ ] ckeditor5-autosave
- [ ] ckeditor5-basic-styles
- [ ] ckeditor5-block-quote
- [ ] ckeditor5-bookmark
- [ ] ckeditor5-ckbox
- [ ] ckeditor5-ckfinder
- [ ] ckeditor5-clipboard
- [ ] ckeditor5-cloud-services
- [ ] ckeditor5-code-block
- [ ] ckeditor5-core
- [ ] ckeditor5-easy-image
- [ ] ckeditor5-editor-balloon
- [ ] ckeditor5-editor-classic
- [ ] ckeditor5-editor-decoupled
- [ ] ckeditor5-editor-inline
- [ ] ckeditor5-editor-multi-root
- [ ] ckeditor5-emoji
- [ ] ckeditor5-engine
- [ ] ckeditor5-enter
- [ ] ckeditor5-essentials
- [ ] ckeditor5-find-and-replace
- [ ] ckeditor5-font
- [ ] ckeditor5-fullscreen
- [ ] ckeditor5-heading
- [ ] ckeditor5-highlight
- [ ] ckeditor5-horizontal-line
- [ ] ckeditor5-html-embed
- [ ] ckeditor5-html-support
- [ ] ckeditor5-icons
- [ ] ckeditor5-image
- [ ] ckeditor5-indent
- [ ] ckeditor5-language
- [ ] ckeditor5-link
- [ ] ckeditor5-list
- [ ] ckeditor5-markdown-gfm
- [ ] ckeditor5-media-embed
- [ ] ckeditor5-mention
- [ ] ckeditor5-minimap
- [ ] ckeditor5-page-break
- [ ] ckeditor5-paragraph
- [ ] ckeditor5-paste-from-office
- [ ] ckeditor5-remove-format
- [ ] ckeditor5-restricted-editing
- [ ] ckeditor5-select-all
- [ ] ckeditor5-show-blocks
- [ ] ckeditor5-source-editing
- [ ] ckeditor5-special-characters
- [ ] ckeditor5-style
- [ ] ckeditor5-table
- [ ] ckeditor5-theme-lark
- [ ] ckeditor5-typing
- [ ] ckeditor5-ui
- [ ] ckeditor5-undo
- [ ] ckeditor5-upload
- [ ] ckeditor5-utils
- [ ] ckeditor5-watchdog
- [ ] ckeditor5-widget
- [ ] ckeditor5-word-count

**CKEditor 5 Commercial:**

- [ ] ckeditor5-ai
- [ ] ckeditor5-case-change
- [ ] ckeditor5-collaboration
- [ ] ckeditor5-collaboration-core
- [ ] ckeditor5-comments
- [ ] ckeditor5-document-outline
- [ ] ckeditor5-email
- [ ] ckeditor5-export-inline-styles
- [ ] ckeditor5-export-pdf
- [ ] ckeditor5-export-word
- [ ] ckeditor5-format-painter
- [ ] ckeditor5-import-word
- [ ] ckeditor5-line-height
- [ ] ckeditor5-list-multi-level
- [ ] ckeditor5-merge-fields
- [ ] ckeditor5-operations-compressor
- [ ] ckeditor5-pagination
- [ ] ckeditor5-paste-from-office-enhanced
- [ ] ckeditor5-premium-features
- [ ] ckeditor5-real-time-collaboration
- [ ] ckeditor5-revision-history
- [ ] ckeditor5-slash-command
- [ ] ckeditor5-source-editing-enhanced
- [ ] ckeditor5-template
- [ ] ckeditor5-track-changes
- [ ] ckeditor5-uploadcare

### PR DESCRIPTION
**Pull request by przemyslaw-zan**
### 🚀 Summary

Replaced test imports with entry point imports wherever possible.

---

### 📌 Related issues

* Closes #18860

---

### 💡 Additional information

*Optional: Notes on decisions, edge cases, or anything helpful for reviewers.*

### CODE REVIEW

File: packages/ckeditor5-link/tests/utils.js [hunk 1]
```diff
@@ -4,16 +4,20 @@
...
+	ViewAttributeElement,
+	ViewContainerElement,
+	ViewText,
+	ModelElement,
+	_setModelData
+} from '@ckeditor/ckeditor5-engine';
+import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+import { BoldEditing } from '@ckeditor/ckeditor5-basic-styles';
+
+// eslint-disable-next-line ckeditor5-rules/allow-imports-only-from-main-package-entry-point
```
Comment (psmyrek) [inline packages/ckeditor5-link/tests/utils.js]: Maybe it would be nice to fix the `ModelSchema` re-export in `index.ts`?

File: packages/ckeditor5-widget/tests/utils.js [hunk 2]
```diff
@@ -22,13 +29,10 @@ import {
 	viewToModelPositionOutsideModelElement,
 	WIDGET_CLASS_NAME
 } from '../src/utils.js';
-import { ViewUIElement } from '@ckeditor/ckeditor5-engine/src/view/uielement.js';
 import { testUtils } from '@ckeditor/ckeditor5-core/tests/_utils/utils.js';
-import { Model } from '@ckeditor/ckeditor5-engine/src/model/model.js';
-import { _setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model.js';
+
+// eslint-disable-next-line ckeditor5-rules/allow-imports-only-from-main-package-entry-point
```
Comment (psmyrek) [inline packages/ckeditor5-widget/tests/utils.js]: Maybe it would be nice to fix the `Mapper` re-export in `index.ts`?

**Review by psmyrek — APPROVED** [review]

**Comment by pomek** [comment]
image

Please, merge master as the final check.
